partiql-spi/src/main/java/org/partiql/spi/value/Datum.java

```diff
@@ -1152,17 +1152,39 @@ static Comparator<Datum> comparator() {
      * operations.
      * </p>
      * @param nullsFirst if true, nulls are ordered before non-null values, otherwise after.
-     * @return the default comparator for {@link Datum}.
+     * @return the default comparator for {@link Datum}. By default, distinguishNullMissing is set to false.
```

Comment:
Since `distinguishNullMissing` is an internal implementation detail. Perhaps you can just say null and missing are treated as equivalent.

```suggestion
     * @return the default comparator for {@link Datum}. By default, null and missing are treated as equivalent.
```